### PR TITLE
Setup extended

### PR DESCRIPTION
**Pull request by AbrahaKahsay**
**In this work:**

- Work section and personal information has added to my portfolio.
- HTML and CSS are used to build the project.
- VS code and Fire fox browsers are used for editing and console.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -134,3 +147,225 @@ body {
...
+  text-align: left;
+  color: #344563;
+}
+
+button {
+  height: 48px;
+  width: 124px;
+  box-sizing: border-box;
+  font-size: 17px;
+  color: #396df2;
```
Comment (lily-coder) [inline style.css:283]: - [ ] [REQUIRED] Please fix the 'See Project' and 'Get My Resume' buttons color to match the original. `background: #FFFFFF;` 😊

File: style.css [hunk 2]
```diff
@@ -134,3 +147,225 @@ body {
 .social-media ul li a {
   color: #505f79;
 }
+
+.works-section {
+  display: flex;
```
Comment (lily-coder) [inline style.css]: - [ ] As stated in the project requirements,
 ### In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

Please use `display: grid;` to layout the elements in this section.😊

File: index.html [hunk 3]
```diff
@@ -38,35 +39,220 @@ <h1 class="name">I’m Abraha Glad to see you!</h1>
...
+          <li class="langguages">
+            Langguages
+            <ul>
+              <li><img src="images/javaScript.svg" alt="javaScript" /></li>
+              <li><img src="images/html.svg" alt="html" /></li>
+              <li><img src="images/css.svg" alt="css" /></li>
+            </ul>
+          </li>
+          <li>Frameworks</li>
+          <li>Skills</li>
```
Comment (lily-coder) [inline index.html]: - [ ] [REQUIRED] As stated in the project requirements, your design should match the original design, please add the right arrow buttons on these two sections _'Frameworks and Skills'_. and add the down arrow button on the _'languages'_ section
- [ ] Remember to include the horizontal lines separating the two segments
### Your Design
stories4

### Original Design

stories3

File: index.html [hunk 4]
```diff
@@ -38,35 +39,220 @@ <h1 class="name">I’m Abraha Glad to see you!</h1>
...
-                ><i class="fa fa-linkedin fa-lg"></i
-              ></a>
-            </li>
-            <li>
-              <a href="#"><i class="fa fa-medium fa-lg"></i></a>
-            </li>
+      </div>
+    </section>
+    <!-- work-section -->
+    <section class="works-section">
```
Comment (lily-coder) [inline index.html:75]: - [ ] [REQUIRED] As stated in the project requirements, you must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma. Please fix the wider margin on the right side of the page to match the original

### Your Version
stories2

### Original Version
stories1

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @AbrahaKahsay , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] [REQUIRED] Using more descriptive titles for your pull request. To learn more on how you can do this, please visit this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) _For example instead of 'setup-extended', you can say 'Mobile Version: Works and About Section'_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (Whoistolu) [reply style.css]: - [ ] [REQUIRED] Kindly implement this, the container of your projects must have a `display: grid;`

File: index.html (hunk 3 above)
Comment (Whoistolu) [reply index.html]: Great job implementing this 👍🏽

File: index.html (hunk 4 above)
Comment (Whoistolu) [reply index.html:75]: - [ ] [REQUIRED] Kindly implement this as well. Please ensure you have equal margins on the right and left side of your cards as it is in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 |

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @AbrahaKahsay ,

You have done a good job implementing some of the recommendations of the previous reviewer however, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Lordkaito — APPROVED** [review]
Hi @AbrahaKahsay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

[OPTIONAL] Please consider avoiding the use of fixed sizes like pixels when making the CSS of your project. This is because it can give you some problems when you want to see it in different browsers, so consider using `rem` or `em` units, or maybe `%` units instead. Please take notes for future projects!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**